src/dmd/nspace.d

```diff
@@ -30,23 +30,27 @@ private enum LOG = false;
  */
 extern (C++) final class Nspace : ScopeDsymbol
 {
-    extern (D) this(const ref Loc loc, Identifier ident, Dsymbols* members)
+    bool ignoreCppSymbols;
```

Comment:
Can you explain the implementation? This is a weird bool, and it's not clear to me from the diff how it works... (I'm not very familiar with DMD)

Reply:
Ohhhh, there's the magic.
Okay, got it. Thanks!

Reply:
All new public symbols should have Ddoc comments.